src/components/CreatePlayer/CreatePlayer.js

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from "react";
+import { Link } from "react-router-dom";
+import nanoid from "nanoid";
+
+import "./createPlayer.css";
+import Button from "../Button";
+import PlayerForm from "../PlayerForm";
+import { GameContext } from "../../contexts/gameContext";
+
+const CreatePlayer = props => {
+  const game = useContext(GameContext);
+  const [players, setPlayers] = game.usePlayers;
+
+  const handlePlayerCreate = player => {
+    const newData = [...players];
+
+    newData.push({
```

Comment:
You have a utility that creates a new player with the id and score already!

Reply:
You want to push a `new Player` here, you're using it as the second argument to `setPlayers` which I don't think would work, did this work for you?